### DISCUSSION TITLE
Set max charge %

### DISCUSSION THREAD

**Discussion by pwwukpw** [alandtse/tesla#180] [Ideas]
I'd like to be able to set max charge level %.

This is so I can automate max charge to 70% for off peak charging but increase to 80% during the day so the car will charge on excess solar generation.

Is this possible or can I make a feature request.

Many thanks

**Comment by alandtse** [comment]
https://github.com/alandtse/tesla/wiki/Modify-Charge-Limit

**Reply by benp09** [reply]
Is it also possible to adjust the amperage via a script?
